src/components/typography/typography.story.js

```diff
@@ -66,15 +66,15 @@ storiesOf('Basics|Typography/Text', module)
         isBold={boolean('bold', false)}
         isInline={boolean('inline', false)}
         isItalic={boolean('italic', false)}
-        tone={select('Text tone', [
-          'none',
-          'primary',
-          'information',
-          'secondary',
-          'positive',
-          'negative',
-          'inverted',
-        ])}
+        tone={select('Text tone', {
+          none: null,
```

Comment:
fixes storybook proptype warning

Reply:
Could it also be fixed by providing a default value to `select`?